### PR TITLE
Extract Advice strings for localization

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: pkg/minikube/extract/extract.go [hunk 1]
```diff
@@ -173,6 +173,11 @@ func inspectFile(e *state) error {
 		return err
 	}
 
+	if strings.HasSuffix(e.filename, "err_map.go") {
```
Comment (tstromberg) [inline pkg/minikube/extract/extract.go]: This seems brittle in the case of the filename changing. Could you make this a compile-time failure if the path changes?

Two ideas: Use the filename as a constant in err_map, or `reflect.PkgPath()`. 

Thoughts?

File: pkg/minikube/extract/extract.go [hunk 2]
```diff
@@ -173,6 +173,11 @@ func inspectFile(e *state) error {
...
+		extractAdvice(file, e)
+		return nil
```
Comment (tstromberg) [inline pkg/minikube/extract/extract.go]: Maybe just `return extractAdvice(...)`?

File: pkg/minikube/extract/extract.go (hunk 1 above)
Comment (sharifelgamal) [reply pkg/minikube/extract/extract.go]: I felt it was easier to explicitly check whether the file existed so I went ahead and did that.

File: pkg/minikube/extract/extract.go (hunk 2 above)
Comment (sharifelgamal) [reply pkg/minikube/extract/extract.go]: yeah. done.

**Pull request merged**